docs/source/gs-processing/usage/distributed-processing-setup.rst

```diff
@@ -104,13 +104,65 @@ the following to build the SageMaker image:
     bash docker/build_gsprocessing_image.sh --environment sagemaker
 
 The above will use the SageMaker-specific Dockerfile of the latest available GSProcessing version,
-build an image and tag it as ``graphstorm-processing-sagemaker:${VERSION}`` where
+build an image and tag it as ``graphstorm-processing-sagemaker:${VERSION}-x86_64`` where
 ``${VERSION}`` will take be the latest available GSProcessing version (e.g. ``0.2.1``).
 
 The script also supports other arguments to customize the image name,
 tag and other aspects of the build. See ``bash docker/build_gsprocessing_image.sh --help``
 for more information.
 
+Support for arm64 architecture
+------------------------------
+
+You might have noticed that we include the image's architecture, ``x86_64`` in the image name.
```

Comment:
I don't think we need this sentence. We just tell them we support both x86_64 and arm64 in pre-processing with EMR serverless.